doc/src/manual/performance-tips.md

```diff
@@ -413,29 +429,31 @@ However, there's one remaining hole: we haven't enforced that `A` has element ty
 perfectly possible to construct an object like this:
 
 ```julia
-julia> b = MyContainer{Int64, UnitRange{Float64}}(1.3:5);
+julia> b = MyContainer{Int64, UnitRange{Float64}}(UnitRange(1.3, 5.0));
```

Comment:
This is a bit awkward now because `1.3:5.0` no longer returns a `UnitRange` but a `StepRangeLen` (which is crazy parameterized).